src/Domain/Creators/TimeDependentOptions/Sphere.cpp

```diff
@@ -213,35 +270,44 @@ void TimeDependentMapOptions::build_maps(
       translation_map_options_.has_value()
           ? std::optional<std::string>{translation_name}
           : std::nullopt,
-      outer_shell_radii.first,
-      outer_shell_radii.second,
+      outer_shell_inner_radius,
+      outer_radius,
       domain::CoordinateMaps::TimeDependent::RotScaleTrans<
           3>::BlockRegion::Inner};
-
-  transition_rot_scale_trans_map_ = RotScaleTransMap{
-      expansion_map_options_.has_value()
-          ? std::optional<std::pair<
-                std::string, std::string>>{{expansion_name,
-                                            expansion_outer_boundary_name}}
-          : std::nullopt,
-      rotation_map_options_.has_value()
-          ? std::optional<std::string>{rotation_name}
-          : std::nullopt,
-      translation_map_options_.has_value()
-          ? std::optional<std::string>{translation_name}
-          : std::nullopt,
-      outer_shell_radii.first,
-      outer_shell_radii.second,
-      domain::CoordinateMaps::TimeDependent::RotScaleTrans<
-          3>::BlockRegion::Transition};
+  if (transition_rot_scale_trans_) {
+    if (radial_partitions.empty()) {
+      ERROR(
+          "Currently at least one radial partition is required to transition "
+          "the RotScaleTrans map to zero in the outer shell.");
+    }
```

Comment:
There's a combination of options here that is likely to lead to confusion because of boundary conditions:
- Has an excision
- Has a shape map
- Has an expansion or translation map
- `transition_rot_scale_trans_` is false

This combination creates a scenario where the outer boundary is moving significantly and thus the only BCs that will work are analytic ones. Our constraint preserving physical BCs won't work. Not sure we should do anything about this since if we have a filled cube and a shape map, the outer boundary can be distorted and there also you have to have special BCs.

Reply:
Yea I don't think we can really prevent these things. Boundary condition classes could try to warn if they assume a static boundary but it is moving or something like that, not sure.